pkg/test/framework/components/policybackend/kube.go

```diff
@@ -84,7 +84,7 @@ spec:
 apiVersion: "config.istio.io/v1alpha2"
 kind: handler
 metadata:
-  name: %s
+  name: {{.namespace}}
```

Comment:
Where is the template variable for namespace being set?

Reply:
This is a wrong change. Tests that use this template were marked as flaky so it was not detected... Fixed now. Thanks!